test/es-module/test-esm-named-exports.js

```diff
@@ -1,13 +0,0 @@
-// Flags: --import ./test/fixtures/es-module-loaders/builtin-named-exports.mjs
-'use strict';
-
-const common = require('../common');
-common.skipIfWorker();
-
-const { readFile, __fromLoader } = require('fs');
-const assert = require('assert');
-
-assert.throws(() => require('../fixtures/es-modules/test-esm-ok.mjs'), { code: 'ERR_REQUIRE_ESM' });
-
-assert(readFile);
-assert(__fromLoader);
```

Comment:
Why is this file deleted?

Reply:
this is replaced by [test/fixtures/es-modules/require-esm-throws-with-loaders.js](https://github.com/nodejs/node/pull/53044/files#diff-ca8e500ac8e629a925b414ca742d27dc471cd158be2d974d0b41208f5f382299) based on https://github.com/nodejs/node/pull/53044#discussion_r1607213480

that one is called with two different configurations from [this test](https://github.com/nodejs/node/pull/53044/files#diff-82a900b610fadd4d760c0af4f8707a3477aa9e28ebe03bfcbb2fcc0cd51947cfR784-R788), once with `--experimental-detect-module` and once without

Reply:
Are you sure? That test was referring to `test/es-module/test-esm-named-exports-detect-module.js` introduced in this PR, not this file `test/es-module/test-esm-named-exports.js`. The `test/es-module/test-esm-named-exports.js` file has a merge conflict from `main` now so it seems like it’s still in use by other features.

Reply:
based on the comment I refactored it such that:
- instead of having [test/es-module/test-esm-named-exports.js](https://github.com/nodejs/node/pull/53044/files#diff-180b76474c1ec59de479f8b03436a27f50131f41ad9bb5c73ac2c9f5b5607e1f)  that has the `// Flags: --import ./test/fixtures/es-module-loaders/builtin-named-exports.mjs` I replaced it by having `describe('should use ESM loaders ', async () => {` [here](https://github.com/nodejs/node/pull/53044/files#diff-82a900b610fadd4d760c0af4f8707a3477aa9e28ebe03bfcbb2fcc0cd51947cfR784) that does the same by loading a fixture from the new file [test/fixtures/es-modules/require-esm-throws-with-loaders.js](https://github.com/nodejs/node/pull/53044/files#diff-ca8e500ac8e629a925b414ca742d27dc471cd158be2d974d0b41208f5f382299)  
- it runs it once as before with [--import](https://github.com/nodejs/node/pull/53044/files#diff-82a900b610fadd4d760c0af4f8707a3477aa9e28ebe03bfcbb2fcc0cd51947cfR792-R794)
- and once more ba adding additionally also the `--experimental-detect-module` from [here](https://github.com/nodejs/node/pull/53044/files#diff-82a900b610fadd4d760c0af4f8707a3477aa9e28ebe03bfcbb2fcc0cd51947cfR787)

So the content of [test/es-module/test-esm-named-exports.js](https://github.com/nodejs/node/pull/53044/files#diff-180b76474c1ec59de479f8b03436a27f50131f41ad9bb5c73ac2c9f5b5607e1f) runs twice with different options from [test/fixtures/es-modules/require-esm-throws-with-loaders.js](https://github.com/nodejs/node/pull/53044/files#diff-ca8e500ac8e629a925b414ca742d27dc471cd158be2d974d0b41208f5f382299)

I rebased the branch and solved the conflicts